ChelseasFieldJournal/src/ModConstants.cs

```diff
@@ -27,15 +38,159 @@ internal static void Init(ModInfo modInfo)
 
             Main_Channel = $"{MOD_ID}:main";
             Config_Channel = $"{MOD_ID}:config";
+            InitializeJournalEntries();
         }
 
         public class EventIDs
         {
             internal static string Config_Reloaded = $"{MOD_ID}:configreloaded";
             internal static string Admin_Send_Config = $"{MOD_ID}:adminsendconfig";
         }
-        
+
         // Place any public enums down here
```

Comment:
We can remove this comment; it's from the template and we know where things should go.